feature/#508 remove pipeline config doc and added sequence doc to scenario config (#538)
Avaiga/taipy-doc: docs/manuals/core/config/scenario-config.md

- _**id**_: The id of a new scenario configuration to be created. This id is **mandatory** and must
  be a unique and valid Python identifier.

> - _**tasks**_: A list of task configuragtions.

Reviewer: "configuragtions" -> "configurations"

> - _**additional_data_nodes**_: A list of additional data node configuragtions.

Reviewer: "configuragtions" -> "configurations"

- _**frequency**_: The recurrence of the scenarios instantiated from this configuration. The scenarios
  are attached to the proper cycles based on this frequency.

> - _**sequences**_: A dictionary of sequence configurations.

Reviewer 2: Sequence is not a configuration object anymore. I propose to change the wording to avoid any confusion:

```suggestion
- _**sequences**_: A dictionary of sequence descriptions.
```

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",
                                         [multiply_task_cfg])
```

In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration already defined.
A `ScenarioConfig^` must always be provided with a list of `TaskConfig^`.

# Adding additional data nodes configurations

Here is a simple example assuming the task configuration `multiply_task_cfg` and the data node configuration
`additional_dn_cfg` have already been created:

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",
                                         [multiply_task_cfg],
                                         [additional_dn_cfg])
```

In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration and
a data node configuration already defined. The provided data node configuration represents additional data.
Additional data node configurations are optional when configuring a `ScenarioConfig^`.

> # Adding sequence configurations

Reviewer 2: ```suggestion
# Adding sequence descriptions
```

We can define sequences through scenario configuration. Here is a simple example assuming the task configuration

> `add_task_cfg_1`, `add_task_cfg_2`, `multiply_task_cfg_1`, `multiply_task_cfg_2` have already been created:

Reviewer 2: ```suggestion
In the scenario configuration, we can describe the sequences to create with the scenario creations. Here is a simple example assuming the task configuration
`add_task_cfg_1`, `add_task_cfg_2`, `multiply_task_cfg_1`, `multiply_task_cfg_2` have already been created:
```

    Note that we cannot use the word "_entity_owner" as a key in the properties as it has been reserved for internal use.

> # From task configs

Reviewer: "configurations", no?

Here is a simple example assuming the task configuration `multiply_task_cfg` has already been created:

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",

>                                          [multiply_task_cfg])

Reviewer: Isn't there a syntax where, if there's only one task, you don't need the array construct?
(please think of the developer and save him time)

Author: @jrobinAV 
We dont have that yet, but that's certainly true! Should we create a ticket for this so we won't forget about it?

Reviewer 2: yes let s create a ticket

```

> In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration already defined.

Reviewer 3: ```suggestion
In this example, we create a scenario configuration `ScenarioConfig^` from a defined task configuration.
```

> A `ScenarioConfig^` must always be provided with a list of `TaskConfig^`.

Reviewer: "... with a list" -> contradicts my proposal (and advice)

# Adding additional data nodes configurations

Here is a simple example assuming the task configuration `multiply_task_cfg` and the data node configuration
`additional_dn_cfg` have already been created:

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",
                                         [multiply_task_cfg],
                                         [additional_dn_cfg])
```

In this example, we create a scenario configuration `ScenarioConfig^` from a task configuration and
a data node configuration already defined. The provided data node configuration represents additional data.
Additional data node configurations are optional when configuring a `ScenarioConfig^`.

# Adding sequence descriptions

> In the scenario configuration, we can describe the sequences to create with the scenario creations.

Reviewer: "to create with the scenario creations." -> "that are part of the configuration."

> In the scenario configuration, we can describe the sequences to create with the scenario creations.

Reviewer 2: ```suggestion
In the scenario configuration, we can describe the sequences to be created with a scenario creation.
```

Here is a simple example assuming the task configuration `add_task_cfg_1`, `add_task_cfg_2`,
`multiply_task_cfg_1`, `multiply_task_cfg_2` have already been created:

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",
                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],

>                                          sequences={"add_sequence": [add_task_cfg_1, add_task_cfg_2,],

Reviewer: I love that syntax!

                                                    "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})
```

> We can also add sequences to your scenario configuration after you have created your scenario configuration.

Reviewer: "We can also add sequences to your" -> "You can also add sequences to your"
(or "We can also add sequences to our" and change the end of the sentence)

Author: will change them all to "we" and "our" to be consistent with other examples

```python linenums="1"
from taipy import Config

> scenario_cfg = Config.configure_scenario("multiply_scenario",

Reviewer: Why those trailing commas before the ')'?
Don't look too nice

Author: ah haha that's true, kinda like a habit when coding at this point. Let me remove them :D

# Or we can add them separately as well
scenario_cfg.add_sequences({"add_sequence": [add_task_cfg_1, add_task_cfg_2,],})
scenario_cfg.add_sequences({"multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})

```

We can also remove sequences after providing them in your scenario configuration.

```python linenums="1"
from taipy import Config

scenario_cfg = Config.configure_scenario("multiply_scenario",
                                         [add_task_cfg_1, add_task_cfg_2, multiply_task_cfg_1, multiply_task_cfg_2],
                                         sequences={"add_sequence": [add_task_cfg_1, add_task_cfg_2,],
                                                    "multiply_sequence": [multiply_task_cfg_1, multiply_task_cfg_2,],})

scenario_cfg.remove_sequences(["add_sequence", "multiply_sequence"])

# Or we can remove a single sequence from a ScenarioConfig by passing only a sequence name to
# ScenarioConfig.remove_sequences
scenario_cfg.remove_sequences("add_sequence")

```
In the small examples above, we create a scenario configuration `ScenarioConfig^` from task configurations and

> defined sequences of that scenario configuration. You can refer back to the sequence concept in

Reviewer: "defined sequences of that scenario configuration" -> "we define sequences for that scenario configuration"

[Sequence concept](../concepts/pipeline.md).

When defining a sequence in a `ScenarioConfig^`, you need to provide a dictionary with the sequence name as the key

> and a list of task configurations that belongs to the sequence as the value. Note that the task configurations of the

Reviewer 3: ```suggestion
and a list of task configurations that belong to the sequence as the value. Note that the task configurations of the
```
